### DISCUSSION TITLE
OTEL is the best commonly used metric specification?

### DISCUSSION THREAD

**Discussion by vincentff7** [open-telemetry/community#1021] [Community Discussions]
**BACKGROUND**: we are building a metric platform, hope to find a common metric specification to format metric to put into kafka. 
thus, any monitor products to produce or consume kafka, can auto recognize the common metric specification in kafka( has already built the translator SDK).  no need to build translator. 
**QUESTION**: opentelemetry OLTP metric specification is the common metric specification? or promethues metric specification is better? or there is no commonly used metric specification?

**Comment by jkowall** [comment]
OTLP is a protocol, OpenTelemetry Metrics is a specification. OpenMetrics is also a specification which is most often implemented in Prometheus. As far as adoption goes, Prometheus is much more adopted today, but there was a lot of work to try to consolidate the two (Otel/OM). I am not certain how close they are in terms of specification, but this is discussed here: https://opentelemetry.io/docs/reference/specification/metrics/datamodel/#prometheus-compatibility. 

OM/Prometheus: https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md
Otel Spec: https://opentelemetry.io/docs/reference/specification/metrics/datamodel/

Prometheus is http text based and Otel is using gRPC in the form of OTLP. gRPC is more efficient, but harder to debug.

Up to you :)
